python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -213,16 +213,18 @@ def _node_type_from_group_spec(
     if is_head:
         # The head node type has no workers because the head is not a worker.
         min_workers = max_workers = 0
+        suspend = False
     else:
         # `minReplicas` and `maxReplicas` are required fields for each workerGroupSpec
         min_workers = group_spec["minReplicas"]
         max_workers = group_spec["maxReplicas"]
+        suspend = group_spec.get("suspend", False)
 
     resources = _get_ray_resources_from_group_spec(group_spec, is_head)
 
     node_type = {
-        "min_workers": min_workers,
-        "max_workers": max_workers,
+        "min_workers": min_workers if not suspend else 0,
+        "max_workers": max_workers if not suspend else 0,
```

Comment:
We need to let the autoscaler know that the suspended worker group should have no workers; otherwise, the autoscaler will keep trying to scale it up.

Reply:
Since the `available_node_types` definition does not provide similar functionality (suspending a node type), the best way to do that is to set its max_workers and min_workers to 0.